slave.mk

```diff
@@ -927,6 +927,9 @@ $(addprefix $(PYTHON_WHEELS_PATH)/, $(SONIC_PYTHON_WHEELS)) : $(PYTHON_WHEELS_PA
 	# Load the target deb from DPKG cache
 	$(call LOAD_CACHE,$*,$@)
 
+	# unset PLATFORM Flag for python wheel generation
+	PLATFORM=
+	export PLATFORM
```

Comment:
Is it possible to have this be applied early in `slave.mk`, so that it applies for all builds, not just Python wheels?

Reply:
It looks like anywhere else unsetting this outside specific targets, have the effect of globally changing the flag even for "make configure" as well. Hence have added only in the specific python wheel targets where this is hitting into the error.